### PR TITLE
adding hex sticker script and logo plus some gitignore changes.

### PR DESCRIPTION
**Pull request by nickreich**

### CODE REVIEW

File: inst/stickers/hubverse-sticker.png
Comment (lshandross) [inline inst/stickers/hubverse-sticker.png:1]: Do we want to change the image displayed in the README to be the hex sticker?

File: inst/stickers/Icon 1000-1000px.png
Comment (lshandross) [inline inst/stickers/Icon 1000-1000px.png:1]: Also, the space in this file name is throwing a warning that it has a "non-portable file name", so it should probably be changed

Comment (nickreich) [reply inst/stickers/Icon 1000-1000px.png:1]: This was fixed in #6

**Comment by lshandross** [comment]
Closing because the changes here were also made in pull request #6, with that one also addressing additional changes requested here.

**Pull request closed without merging**